aws-auth-cognito/src/test/java/com/amplifyframework/auth/cognito/featuretest/generators/SerializationTools.kt

```diff
@@ -183,6 +185,15 @@ fun Any?.toJsonElement(): JsonElement {
     }
 }
 
+fun AWSCognitoAuthSignInOptions.toJsonElement(): JsonElement {
```

Comment:
Is this necessary? gsonBased or ReflectionBased Serializers should handle these. 
Reason for having them is to have to have as less custom serializers as possible.

Reply:
Removed.